src/lib/markbind/src/utils/nunjuckUtils.js

```diff
@@ -1,73 +1,13 @@
 const nunjucks = require('nunjucks');
 const dateFilter = require('../lib/nunjucks-extensions/nunjucks-date');
 
-const commonEnv = nunjucks.configure().addFilter('date', dateFilter);
 const unescapedEnv = nunjucks.configure({ autoescape: false }).addFilter('date', dateFilter);
 
-const START_ESCAPE_STR = '{% raw %}';
-const END_ESCAPE_STR = '{% endraw %}';
-const RAW_TAG_REGEX = new RegExp('{% *(end)?raw *%}', 'g');
-
-/**
- * Pads the outermost {% raw %} {% endraw %} pairs with {% raw %} {% endraw %} again.
- * This allows variables and other nunjuck syntax inside {% raw %} {% endraw %} tags
- * to be ignored by nunjucks until the final renderString call.
- */
-function preEscapeRawTags(pageData) {
```

Comment:
I guess the real solution to #762 was to make nunjucks a single pass, removing the need for preescaping `raw/endraw`. A little unfortunate that the work done here is removed almost as soon as it was introduced #1220 #1049, but we could always revert it in the future if there's ever a need for multiple nunjucks passes.

Reply:
@crphang could I get your opinion on this?